util/neo-nrve-eventhandler.py

```diff
@@ -109,7 +118,7 @@ def sc_notify(self, event):
         # The event payload list has at least one element. As developer of the smart contract
         # you should know what data-type is in the bytes, and how to decode it. In this example,
         # it's just a string, so we decode it with utf-8:
-        event_type = event.event_payload[0].decode("utf-8")
+        event_type = payload[0].decode("utf-8")
```

Comment:
Had to do the following here: `event_type = payload[0].Value.decode("utf-8")`

Reply:
Probably need to update all the places where we look at 'payload[x]' values?

Reply:
Yup, I think you've got it! Sorry to throw an incomplete PR at you. I'll get it fixed! Thanks 👍